README.md

```diff
@@ -94,7 +94,9 @@ This creates a Podfile, which will hold all your dependencies in one place. Afte
 
     pod install
 
-to install the libraries and include them as part of a workspace which also holds your own project. It is generally [recommended to commit the installed dependencies to your own repo][committing-pods], instead of relying on having each developer running `pod install` after a fresh checkout.
+to install the libraries and include them as part of a workspace which also holds your own project.
+
+It is generally [recommended to commit the installed dependencies to your own repo][committing-pods], instead of relying on having each developer running `pod install` after a fresh checkout. (This is also the [CocoaPods-endorsed][committing-pods-cocoapods] way of doing things.)
```

Comment:
This sounds like a repetition to me: First 'It is generally recommended' followed by 'This is also the CocoaPods-endorsed way'.
Maybe we could say "We recommend..." And then maybe give a reason as to why?
What would these reasons be?
1. Independence of the upstream availability.
2. Ease of branch switching.
...?

Reply:
Hm, maybe that's better. But we also don't want to clutter the CocoaPods section with this stuff. Maybe just "we recommend" and then state the two links as motivation?

Reply:
Sounds good.